Screenbox/Pages/ArtistDetailsPage.xaml

```diff
@@ -283,9 +339,10 @@
                         <Setter Target="ProfilePicture.Height" Value="140" />
                         <Setter Target="ProfilePictureIcon.FontSize" Value="52" />
                         <Setter Target="Subtext.Style" Value="{StaticResource BodyTextBlockStyle}" />
+                        <Setter Target="Subtext.TextWrapping" Value="NoWrap" />
                         <Setter Target="ButtonPanel.Margin" Value="0" />
                         <Setter Target="ItemList.Padding" Value="{StaticResource PageContentMarginMinimal}" />
-                        <!-- <Setter Target="ItemsStackPanel.GroupHeaderPlacement" Value="Top" /> -->
+                        <Setter Target="ItemsStackPanel.GroupHeaderPlacement" Value="Top" />
```

Comment:
This is causing the enter the page in minimal state regression.

Reply:
Found the problem. The page doesn't automatically set a visual state on page enter. Manually set the state on page loaded.